src/users.jrl

```diff
@@ -28,3 +26,24 @@ p({
   "onboarded":true,
   "loginEnabled":true
 })
+p({
+  class:"foam.nanos.auth.User",
+  id:95250647,
+  spid:"foam",
+  userName:"anonymous-foam",
+  firstName:"Anonymous",
+  lastName:"Foam",
+  email:"anonymous-foam@foam.foam",
+  emailVerified:true,
+  password:"WsJlIJDGGGQ=:cnE+tRNtF/yhOheF2POyNuocgKTZDVyXia7dITQFUEVPS9hpuQqp8lVTTm2iQQXbYk/8cBUCOV/prsUkO793Sg==",
```

Comment:
should the anonymous user have a password?  Seems safer if we have no password and somehow disable password login for the anonymous user

Reply:
bump